CMakeLists.txt

```diff
@@ -391,6 +391,9 @@ if(opencoarrays_aware_compiler)
   add_mpi_test(duplicate_syncimages 8 ${tests_root}/unit/sync/duplicate_syncimages)
   add_mpi_test(co_reduce 4 ${tests_root}/unit/collectives/co_reduce_test)
   add_mpi_test(syncimages_status 32 ${tests_root}/unit/sync/syncimages_status)
+  add_mpi_test(syncimages_ring_np3 3 ${tests_root}/unit/sync/syncimages_ring)
+  add_mpi_test(syncimages_ring_np13 13 ${tests_root}/unit/sync/syncimages_ring)
+  add_mpi_test(syncimages_ring_np23 23 ${tests_root}/unit/sync/syncimages_ring)
```

Comment:
Arbitrary, in fact, just by adding a 1 and a 2 before the 3 of the initial test. I did not want to have a power of two here.